### PR TITLE
Add activation form

### PR DESCRIPTION
**Pull request by Karkavin**

### CODE REVIEW

**Comment by Karkavin** [comment]
Refactoring has already finished

File: angio-frontend/src/modules/common/helpers/commonEvent.ts [hunk 1]
```diff
@@ -3,4 +3,5 @@ export enum CommonEvent {
     INPUT = 'input',
     UPDATE = 'update',
     SEND_FORM = 'send-form',
+    ACTIVATE_USER = 'activate-user',
```
Comment (nikon-petr) [inline angio-frontend/src/modules/common/helpers/commonEvent.ts]: Use `SEND_FORM`

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    @Component({
+        components: {BaseSingleFormContainer}
+    })
+    export default class AccountActivationForm extends Vue {
+
+        @Prop()
+        public readonly fetching!: boolean;
+
+        @Prop()
+        public readonly step!: number;
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Use `enum` for step field

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+        @Prop()
+        public readonly displayError!: boolean;
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Pass error messages array from view component and show it if not empty.

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                    <span>{{ $t('user.component.activation.activationSuccessStep.title') }}</span>
+                    <v-spacer></v-spacer>
+                </v-card-title>
+                <v-card-text class="subheading font-weight-light">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.content') }}</span>
+                </v-card-text>
+            </v-window-item>
+        </v-window>
+
+        <v-divider></v-divider>
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Remove divider

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-expand-transition>
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Create separate component for form messages with prop: `errorMessages: string[]`

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 6]
```diff
@@ -0,0 +1,174 @@
+<template>
+    <v-card>
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Add paddings like the `BaseSingleFormContainer`

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+                    v-bind:loading="fetching"
+                    form="activation-form"
+                    type="submit"
+                    color="primary"
+                    flat
+                    round
+            >
+                {{ $t('user.component.activation.activationFormStep.button.send') }}
+            </v-btn>
+            <v-btn
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Move to center of second step

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+                </v-card-text>
+            </v-window-item>
+
+            <v-window-item v-bind:value="2">
+                <v-card-title class="headline font-weight-regular justify-space-between">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.title') }}</span>
+                    <v-spacer></v-spacer>
+                </v-card-title>
+                <v-card-text class="subheading font-weight-light">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.content') }}</span>
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Replace with icon (name: `done`, size: `x-large`) and short text description (less than 50 characters). Icon [doc](https://vuetifyjs.com/ru/components/icons).

File: angio-frontend/src/modules/common/helpers/commonEvent.ts (hunk 1 above)
Comment (Karkavin) [reply angio-frontend/src/modules/common/helpers/commonEvent.ts]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 2 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 4 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 6 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 7 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 8 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 5 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 3 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done. Good idea. Apply it to LoginForm and ActivationForm.

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 7 above)
Comment (nikon-petr) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Not to center of bottom of second step but to center bellow icon and text like "home" button on the 404 page.

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 2 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue (hunk 7 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

**Comment by Karkavin** [comment]
PR has already updated

**Pull request merged**